content/zh/docs/tasks/configure-pod-container/security-context.md

```diff
@@ -0,0 +1,690 @@
+---
+title: 为 Pod 或容器配置安全性上下文
+content_type: task
+weight: 80
+---
+<!--
+reviewers:
+- erictune
+- mikedanese
+- thockin
+title: Configure a Security Context for a Pod or Container
+content_type: task
+weight: 80
+-->
+
+<!-- overview -->
+
+<!--
+A security context defines privilege and access control settings for
+a Pod or Container. Security context settings include, but are not limited to:
+
+* Discretionary Access Control: Permission to access an object, like a file, is based on
+  [user ID (UID) and group ID (GID)](https://wiki.archlinux.org/index.php/users_and_groups).
+* [Security Enhanced Linux (SELinux)](https://en.wikipedia.org/wiki/Security-Enhanced_Linux): Objects are assigned security labels.
+* Running as privileged or unprivileged.
+* [Linux Capabilities](https://linux-audit.com/linux-capabilities-hardening-linux-binaries-by-removing-setuid/): 
+  Give a process some privileges, but not all the privileges of the root user.
+-->
+安全上下文（Security Context）定义 Pod 或 Container 的特权与访问控制设置。
+安全上下文包括但不限于：
+
+* 自主访问控制（Discretionary Access Control）：基于
+  [用户 ID（UID）和组 ID（GID）](https://wiki.archlinux.org/index.php/users_and_groups).
+  来判定对对象（例如文件）的访问权限。
+* [安全性增强的 Linux（SELinux）](https://en.wikipedia.org/wiki/Security-Enhanced_Linux)：
```

Comment:
Is https://zh.wikipedia.org/wiki/%E5%AE%89%E5%85%A8%E5%A2%9E%E5%BC%BA%E5%BC%8FLinux sufficiently good to link to? If not, it makes sense to link to https://en.wikipedia.org/wiki/Security-Enhanced_Linux.

Reply:
Good point. Will verify if the Chinese version of wikipedia page is available.